### PR TITLE
Fixes #12308 & Fixes #12178

### LINKED ISSUE #12178: Mice can place stuff in disposals
(WEB REPORT BY: jeroen52 REMOTE: 172.93.109.202:7777)

> Revision (Should be above if you're viewing this from ingame!)
> 02a01197183ab1b5eb1b5655d68eb7944e7f6d63
> 
> General description of the issue
> A mouse can put items in disposals.
> 
> What you expected to happen
> The mouse not putting something in disposals
> 
> What actually happened
> mouse (794) places the red suit into the the disposal unit. 
> 
> Steps to reproduce if possible
> No clue, I assume you become a mouse.

### LINKED ISSUE #12308: Some mobs still have hands when they probably shouldn't
Mobs like mice that shouldn't have hands still do and can use them (kinda).  It means anything that uses `put_in_hands()` (and probably anything that only works on hand checks) works on mice/etc.

So you can, for example, steal a magazine out of a gun.  You can't drop it though, you have to put it into something like a box or back into the gun.

### PR DESCRIPTION
**Pull request by ghost**
Fixes #12308 & Fixes #12178

### CODE REVIEW

File: code/modules/mob/living/simple_animal/hostile/slime.dm [hunk 1]
```diff
@@ -22,6 +22,7 @@
 	meat_type = null
 
 	var/colour = "grey"
+	held_items = list()
```
Comment (Intigracy) [inline code/modules/mob/living/simple_animal/hostile/slime.dm:25]: rip slimes pulling each other around

**Comment by Intigracy** [comment]
Could have fixed the nulling lightbulbs part of #12774 with this too.

**Comment by ghost** [comment]
@Intigracy Don't think I did, didn't know there was a subtype of monkies without hands. Would be an easy fix though

**Comment by Intigracy** [comment]
I meant you could have, not that this possibly did.

**Pull request merged**